ui/zenoedit/launch/ztcpserver.cpp

```diff
@@ -0,0 +1,112 @@
+#include "ztcpserver.h"
```

Comment:
Please guard it with `#ifdef ZENO_MULTIPROCESS` to make sure single-process still work as always.